src/main/java/org/terasology/nui/StuffLengthsScreen.java

```diff
@@ -0,0 +1,44 @@
+	private int infoIndex = 0;
+
+	@Override
+	public void initialise() {
+		infoArea = find("infoArea", UIText.class);
+		updateInfoButton = find("updateInfoButton", UIButton.class);
+
+		if (updateInfoButton != null) {
+			updateInfoButton.subscribe(button -> {
+				infoArea.setText(infoItems[infoIndex]);
+				if (infoIndex > infoItems.length() - 2)
```

Comment:
We always use brackets, even if the if only has one statement. Same goes for the else clause below.